pkg/agent/openflow/client.go

```diff
@@ -596,10 +606,19 @@ func (c *client) InstallClusterServiceFlows() error {
 	if c.IsIPv4Enabled() {
 		flows = append(flows, c.serviceHairpinResponseDNATFlow(binding.ProtocolIP))
 		flows = append(flows, c.serviceLBBypassFlows(binding.ProtocolIP)...)
+		flows = append(flows, c.l3FwdServiceDefaultFlowsViaGW(binding.ProtocolIP, cookie.Service)...)
+		if c.enableProxyFull {
+			flows = append(flows, c.serviceHairpinRegSetFlows(binding.ProtocolIP))
```

Comment:
Add comments to explain why these two flows are added.

Reply:
// The output interface of a packet is the same as where it is from,  the action of the packet should be
// IN_PORT, rather than output. When a packet of Service is from Antrea gateway and its Endpoint is on host
// network, it needs hairpin mark (by setting a register, it will be matched at table L2ForwardingOutTable #110).
flows = append(flows, c.serviceHairpinRegSetFlows(binding.ProtocolIP))
// This flow is used to response to the ARP request of Service virtual IP.
flows = append(flows, c.arpResponderFlow(config.ServiceVirtualIPv4.To4(), cookie.Service))